EIPS/eip-draft_tx_expir.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: <to be assigned>
+title: Transaction Expiration
+description: Proposes a Transaction Expiration Block for every transaction.
+author: Lucas Vinzon<lucasvinzon@gmail.com>
+discussions-to: <URL>
```

Comment:
You'll need an [Ethereum Magicians](https://ethereum-magicians.org) thread for discussions.